lms/static/js/student_account/views/account_settings_factory.js

```diff
@@ -187,9 +188,83 @@
                             })
                         }
                     ]
+                },
+                {
+                    title: gettext('Social Links'),
```

Comment:
It's not as straight-forward as it should appear - for now I will leave as is, just to match the other sections. I agree though that this is something we should fix up down the road.